presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -111,7 +111,7 @@ export class QueryListItem extends React.Component {
                     <span className="glyphicon glyphicon-fire" style={GLYPHICON_HIGHLIGHT}/>&nbsp;&nbsp;
                     {query.queryStats.peakUserMemoryReservation}
                 </span>
-                <span className="tinystat" data-toggle="tooltip" data-placement="top" title="Cumulative memory">
+                <span className="tinystat" data-toggle="tooltip" data-placement="top" title="Cumulative user memory">
                     <span className="glyphicon glyphicon-equalizer" style={GLYPHICON_HIGHLIGHT}/>&nbsp;&nbsp;
                     {formatDataSizeBytes(query.queryStats.cumulativeUserMemory)}
```

Comment:
Did you intend to change this line?

Reply:
Also, regarding cmt msg:
> `Fix Cumulative User Memory UI presentationThe memory value on the query list view was showing raw value
(miliseconds) while the query details view was showing seconds, this
lead to questions from confused users. Changed to show seconds in both
views.`

This isn't "seconds". This is "bytes-seconds", or however this should be called in English.